src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeParameterInfo.cs

```diff
@@ -394,6 +405,7 @@ private static decimal GetRawDecimalConstant(CustomAttributeData attr)
         {
             Debug.Assert(attr.Constructor.DeclaringType == typeof(DecimalConstantAttribute));
 
+            // FIXME: Is this even possible?
```

Comment:
The `Debug.Fail` says below says that it is not possible.

Reply:
I was thinking if we could remove the loop all together

Reply:
Yes, that sounds good to me.